scripts/global_ens/stats/exevs_global_ens_atmos_grid2obs_stats.sh

```diff
@@ -46,7 +46,17 @@ export verif_case=$3
 
 
 if [ $ens = all ] || [ $ens = gefs ] || [ $ens = cmce ] || [ $ens = naefs ] || [ $ens = ecme ] ; then 	
- $USHevs/global_ens/evs_global_ens_atmos_${verif_case}.sh $ens  
+    if [ ! -s ${COMIN}.${VDATE}/gefs/gfs.t00z.prepbufr.f00.nc ] ; then
+       export subject="PREPBUFR data file missing "
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
@BinbinZhou-NOAA  this line (51) needs to be removed now that the variable is exported from the ecf script.